client/rest-high-level/src/test/java/org/elasticsearch/client/indexlifecycle/IndexExplainResponseTests.java

```diff
@@ -194,4 +113,10 @@ public String toString() {
         }
     }
 
+    @Override
+    protected NamedXContentRegistry xContentRegistry() {
+        List<NamedXContentRegistry.Entry> entries = new ArrayList<>(ClusterModule.getNamedXWriteables());
+        entries.add(new NamedXContentRegistry.Entry(LifecycleAction.class, new ParseField(DeleteAction.NAME), DeleteAction::parse));
```

Comment:
We could probably do

```java
new NamedXContentRegistry(new IndexLifecycle(Settings.EMPTY).getNamedXContent())
```

so that we don't hardcode the Delete action here

Reply:
IndexLifecycle is not available to the client

Reply:
Oh of course, if only Github didn't truncate the path so much... :)